modules/kubernetes-addons/velero/variables.tf

```diff
@@ -0,0 +1,38 @@
+variable "helm_config" {
+  type        = any
+  description = "Helm provider config for the kube-state-metrics"
+  default     = {}
+}
+
+variable "manage_via_gitops" {
+  type        = bool
+  default     = false
+  description = "Determines if the add-on should be managed via GitOps."
+}
+
+variable "irsa_permissions_boundary" {
+  type        = string
+  default     = ""
+  description = "IAM Policy ARN for IRSA IAM role permissions boundary"
+}
```

Comment:
This goes in `addon_context` along with `irsa_iam_role_path`. Please see other addon examples.